message_sender/tests.py

```diff
@@ -968,47 +1014,6 @@ def test_create_inbound_data_with_concurrency_limiter(self):
         self.assertEqual(d.transport_type, "voice")
         self.assertEqual(d.helper_metadata, {"session_event": "close"})
 
-    @responses.activate
```

Comment:
Yes, from what I could see, these tests are exactly the same as other tests, so I removed them. Like this test looks the same as `test_create_inbound_data_with_channel_junebug`, unless I missed something.